### PR TITLE
 Add --update-list CLI option

### PR DESCRIPTION
**Pull request by dreispt**
CLI option to update the module list.
Using it allows to directly install newly added modules without going to the web GUI or updating the base module.

### CODE REVIEW

**Comment by guewen** [comment]
Great!

**Comment by jgrandguillaume** [comment]
:+1:

File: openerp/tools/config.py [hunk 1]
```diff
@@ -107,7 +107,8 @@ def __init__(self, fname=None):
         # Not exposed in the configuration file.
         self.blacklist_for_save = set([
             'publisher_warranty_url', 'load_language', 'root_path',
-            'init', 'save', 'config', 'update', 'stop_after_init'
+            'init', 'save', 'config', 'update', 'stop_after_init',
+            'update-list'
```
Comment (rco-odoo) [inline openerp/tools/config.py]: beware of typo: `-` instead of `_`

**Comment by dreispt** [comment]
Fixed!

File: addons/base_action_rule/base_action_rule.py [hunk 2]
```diff
@@ -217,6 +217,7 @@ def write(self, cr, uid, ids, vals, context=None, **kwargs):
                 action_dom = [('model', '=', self._name),
                               ('kind', 'in', ['on_write', 'on_create_or_write'])]
                 action_ids = action_model.search(cr, uid, action_dom, context=context)
+                #import pudb; pudb.set_trace()
```
Comment (matrixise) [inline addons/base_action_rule/base_action_rule.py]: could you remove this line, thanks

File: addons/website_hr_recruitment/controllers/main.py [hunk 3]
```diff
@@ -92,6 +92,8 @@ def jobs_apply(self, job):
 
     @http.route('/jobs/thankyou', methods=['POST'], type='http', auth="public", website=True)
     def jobs_thankyou(self, **post):
+        import pprint; pprint.pprint(post)
+        import pudb; pudb.set_trace()
```
Comment (matrixise) [inline addons/website_hr_recruitment/controllers/main.py]: ??

**Comment by dreispt** [comment]
Sorry, something went wildly with this PR.
Reopened it at https://github.com/odoo/odoo/pull/9133

**Pull request closed without merging**